nubison_model/Service.py

```diff
@@ -30,10 +31,16 @@ def load_nubison_model(
         if not mlflow_model_uri:
             raise RuntimeError("MLflow model URI is not set")
         set_tracking_uri(mlflow_tracking_uri)
-        mlflow_model = load_model(
-            model_uri=mlflow_model_uri, model_config={"initialize": initialize}
+        mlflow_model = load_model(model_uri=mlflow_model_uri)
+
+        nubison_mlflow_model = cast(
```

Comment:
I'm curious about the reason why the cast library was used to display a warning. Also, why was the get_nubison_model() function removed?

Reply:
`mlflow_model.unwrap_python_model()` returns Any, but it should actually return a `NubisonMLFlowModel`. I placed it in `cast()` to ensure clarity.
This function calls `get_nubison_model()` a couple of lines below. I separated this line into the following lines so that I can use both `prepare_artifacts()` and `get_nubison_model()`.